docs/advanced/exceptions.rst

```diff
@@ -79,6 +79,20 @@ This call creates a Python exception class with the name ``PyExp`` in the given
 module and automatically converts any encountered exceptions of type ``CppExp``
 into Python exceptions of type ``PyExp``.
 
+It is possible to specify base class for the exception using the third 
+parameter, a pointer to `PyObject`:
```

Comment:
I know this is currently correct, but do we have any idea why this isn't a `handle`?

Reply:
No idea, first time contributing to pybind.

Reply:
No problem. Let's see if @bstaletic or anyone else knows! :-)

Reply:
Nope, no clue other than "the source says so".

Reply:
OK, let's maybe change this in a different PR, then? (Actually, we might even change this to `py::type`, @henryiii?)